django/db/models/base.py

```diff
@@ -75,6 +75,18 @@ def __new__(cls, name, bases, attrs, **kwargs):
         classcell = attrs.pop('__classcell__', None)
         if classcell is not None:
             new_attrs['__classcell__'] = classcell
+        # Fix related to bug #28198.
```

Comment:
We generally try to describe the issue sufficient so that a ticket reference isn't needed.